src/Components/SystemDetail/RecommendationsTable.scss

```diff
@@ -2,15 +2,35 @@
   tr {
     --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
   }
-  .newline {
-    white-space: pre-line;
-  }
-  .tab {
-    white-space: pre-wrap;
-  }
   .recs-table-empty {
     .pf-c-empty-state__icon {
       fill: var(--pf-global--success-color--100);
     }
   }
 }
+.newline {
+  white-space: pre-line;
+}
+.tab {
+  white-space: pre-wrap;
+}
+Th {
+  font-weight: normal;
+}
+strong {
+  padding: 12px;
+}
+.table-border-top {
+  border-top: 1px solid #d9d9d9;
```

Comment:
@r14chandra I understand we adding this to add the top border but I observed that the space between top border and bottom is not consistent. Can we fix it please? attaching the screenshot for reference
![image](https://user-images.githubusercontent.com/5928530/161260140-7f822389-e53e-42b1-9ac8-9ea12ab5553c.png)